sbom/sbom.proto

```diff
@@ -36,6 +36,30 @@ enum Status {
   // are missing, preventing the BOM from being usable or reliable. A failed
   // status typically requires a thorough review and reprocessing of the BOM.
   STATUS_FAILED = 3;
+
+  // STARTED indicates that the SBOM scan was initialized. It is used for tracking
+  // progress of a scan over time.
+  STATUS_STARTED = 4;
+}
+
+// ProgressMessage 
+message ProgressMessage {
```

Comment:
I do not think we really need this. We could return the sbom message. timestamp is already part of it and asset has the external id included, status too.